server/server.js

```diff
@@ -39,4 +39,5 @@ VotesCollection.after.remove(function afterRemove(userId, vote) {
     userId && collection && collection.update(vote.linkedObjectId, { $inc: { voteScore: -vote.direction } });
 });
 
+export { VoteableModel } from '../common/voteable-model.js';
```

Comment:
Something doesn't seem right here. We shouldn't have 2 named exports of the same name. I'll have to check into this and see if there's an underlying issue.

Reply:
This is only on server. The problem is was that `VoteableModel` has been exported on client, but not on server.